### DISCUSSION TITLE
EPG URL to match playlists.

### DISCUSSION THREAD

**Discussion by Irishhood** [iptv-org/community#560] [Q&A] [answered]
EPG URL to match playlists. For example. I am using 🇺🇸 United States	1811 https://iptv-org.github.io/iptv/countries/us.m3u but cannot find the EPG for it. I am new. Can someone please help me?

**Comment by Animenosekai** [comment]
You can go to [iptv-org/epg](https://github.com/iptv-org/epg) and follow instructions to download EPGs there.

If you want an example of a way it could be used, this is exactly I'm doing for the repo [Animenosekai/japanterebi-xmltv](https://github.com/Animenosekai/japanterebi-xmltv).

You can use the different scripts there are or even fork it and change the `.github/workflows/update.yaml` file to match your needs (I guess using the United States for the country filtering and such)

**Comment by BellezaEmporium** [comment, marked as answer]
We do not serve EPG files anymore due to certain restrictions given by GitHub over the Actions system.

Best way is to go to iptv-org/epg and generate the different playlists for your channels, based on the available scripts.
